frontend/src/components/cluster-dashboard/ClusterDashboard.tsx

```diff
@@ -1,17 +1,47 @@
-import React from 'react';
+import React, { useState, useEffect, Dispatch, SetStateAction } from 'react';
+import { Grid } from '@material-ui/core';
+import axios from 'axios';
+
+import { akkaClusterProps } from './../../assets/properties/akkaClusterProps';
 import './ClusterDashboard.scss';
-import { Grid, Paper } from '@material-ui/core';
+import ClusterListView from './cluster-list-view/ClusterListView';
+import { Cluster } from './Cluster.model';
+import SimpleSnackBar from '../shared/simple-snack-bar/SimpleSnackBar';
 
 const ClusterDashboard: React.FC = () => {
+  
+  const [cluster, setCluster]: [Cluster, Dispatch<SetStateAction<Cluster>>] = useState(new Cluster());
+  const [openSnackBar, setOpenSnackBar] = React.useState(false);
+  const [snackBarMessage, setSnackBarMessage] = React.useState("");
+  
+  const loadClusterData = (): void => {
+    const akkaGetClusterMembersUrl = akkaClusterProps["akka.management.url"] + "/cluster/members";
```

Comment:
Instead of concatonating strings, it's best practice to use template literals. Try this:

`const akkaGetClusterMembersUrl = `${akkaClusterProps["akka.management.url"]}/cluster/members``;